src/nvim/api/vim.c

```diff
@@ -252,17 +252,23 @@ void nvim_feedkeys(String keys, String mode, Boolean escape_ks)
   for (size_t i = 0; i < mode.size; i++) {
     switch (mode.data[i]) {
     case 'n':
-      remap = false; break;
+      remap = false;
+      break;
     case 'm':
-      remap = true; break;
+      remap = true;
+      break;
     case 't':
-      typed = true; break;
+      typed = true;
+      break;
     case 'i':
-      insert = true; break;
+      insert = true;
+      break;
     case 'x':
-      execute = true; break;
+      execute = true;
+      break;
     case '!':
-      dangerous = true; break;
+      dangerous = true;
+      break;
```

Comment:
What lint warnings does this fix?

Reply:
It fixes `whitespace/newline`. See: https://raw.githubusercontent.com/neovim/doc/gh-pages/reports/clint/errors.json 

They have `verbosity=0` which is why you might not have seen them.

We don't have to go with this change. I'm not sure if I'm a fan of it. I'm just trying to close out the remaining errors.

Reply:
If switch-case formatting is covered by uncrustify, let's remove the clint rule. cc @dundargoc

Reply:
Uncrustify does cover this, it doesn't allow `case 'n': stmt; break;` as it requires a newline after `:`.

Reply:
Raised #20619

Reply:
There was a specific reason why I didn't add this fix in uncrustify, but I dont remember why. Can investigate.